pages/common/xml-list.md

```diff
@@ -0,0 +1,16 @@
+# xml-list
+
+> List a directory as XML.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- List the current directory to an XML file:
```

Comment:
Perhaps this will read better?
```suggestion
- List the items in the current directory as an XML document:
```

Reply:
- List the current directory's contents to an XML document: